packages/volto-iframe-block/src/components/Edit.jsx

```diff
@@ -30,6 +31,7 @@ const IframeEdit = (props) => {
   const { data, intl } = props;
   const [url, setUrl] = useState('');
   const iframeRef = useRef(null);
+  const siteData = useSelector((state) => state.site.data);
```

Comment:
Let's be really careful in case someone uses this with an old version of Volto from before we added the site data.
```suggestion
  const siteData = useSelector((state) => state.site?.data);
```